### PR TITLE
Duplicate reviewers email ids discarded

### PR DESCRIPTION
**Pull request by roshan-sy**
Please make sure the code is following contribution guidelines in CONTRIBUTING.md

 - [ ] : This PR has a corresponding issue open in the Repository.
 - [ ] : Approach is signed off on the issue.

Fixes: #1118 

`reviewers = list(set(x.lower() for x in reviewers))`
using above code snippet to discard duplicate reviewer
As reviewers object is list of email Ids, it is safe to do apply `.lower()`

### CODE REVIEW

**Review by gauravsaralMs — COMMENTED** [review]
can we add a unit test for this change ?

File: azure-devops/azext_devops/test/repos/test_pull_request.py [hunk 1]
```diff
@@ -134,6 +134,38 @@ def test_create_pull_request(self):
...
+        self.mock_create_PR.return_value.id = test_pr_id
+        self.mock_resolve_reviewers_as_ids.return_value = ['id1']
+
+        response = create_pull_request(project = self._TEST_PROJECT_NAME,
+        repository = self._TEST_REPOSITORY_NAME,
+        source_branch = self._TEST_SOURCE_BRANCH,
+        target_branch = self._TEST_TARGET_BRANCH,
+        title = self._TEST_PR_TITLE,
+        description = self._TEST_PR_DESCRIPTION,    
+        reviewers = ['a@b.com','A@b.com'],
```
Comment (gauravsaralMs) [inline azure-devops/azext_devops/test/repos/test_pull_request.py:152]: we don't have an assert for this ?

Comment (gauravsaralMs) [reply azure-devops/azext_devops/test/repos/test_pull_request.py:152]: @roshan-sy : we can evaluate if https://github.com/Azure/azure-devops-cli-extension/blob/c5df216e195ddd6fc085298abd6bb39bdc868ca5/azure-devops/azext_devops/test/repos/test_pull_request.py#L90 is called only with one reviewer

Comment (roshan-sy) [reply azure-devops/azext_devops/test/repos/test_pull_request.py:155]: image

**Review by gauravsaralMs — APPROVED** [review]

**Pull request merged**